tests/installation/select_patterns.pm

```diff
@@ -26,6 +26,9 @@ sub accept3rdparty {
 sub movedownelseend {
     my $ret = wait_screen_change {
         send_key 'down';
+        if (check_screen('pattern-too-low', 5)) {
+            assert_and_click('pattern-too-low', 'left', 30, 0.5);
+        }
```

Comment:
beside that - having a check_screen within a wait_screen_change sounds like a bad idea